OpenSim/Common/ComponentConnector.h

```diff
@@ -133,9 +134,18 @@ class OSIMCOMMON_API AbstractConnector : public Object {
     /** Disconnect this Connector from the connectee object */
     virtual void disconnect() = 0;
 
+protected:
+    const Component& getOwner() const { return _owner.getRef(); }
+    void setOwner(const Component& o) { _owner.reset(&o); }
```

Comment:
From the users point of view I prefer the getter to return a const ref and then for consistency I would like the setter to take a const ref. I think most, if not all, of our get<something> methods return a const reference.